packages/wallets/bitkeep/src/adapter.ts

```diff
@@ -13,37 +15,26 @@ import {
+    name = BitgetWalletName;
+    url = WalletInfo?.homepage;
+    icon = WalletInfo?.logolist?.base64;
```

Comment:
These cannot be changed this way, because these imports cannot be used except as types. Please see the below comment, and inline the new image. Please also include the SVG file, as it may need to be compressed.